docs/src/self-hosting.md

```diff
@@ -32,8 +32,22 @@ _Note: you might find some rough edges at this time, we are very interested to s
     - choose your server
     - choose a domain name that karrot will be deployed under: this is what we refer to as `karrot app name`
 2. `abra app config <karrot app name>` see [configuration options](#configuration-options) below
-3. `abra app deploy <karrot app name>`
-4. Wait for a few minutes while karrot initializes and then you can start creating accounts!
+3. Set all the secrets, even the ones you don't want to use:
+   - `abra app secret list <karrot app name>` for listing all secrets
+   - `abra app secret insert <karrot app name> <secret name> <version> "<secret value>"` for inserting a secret
+   - `abra app secret generate <karrot app name> <secret name> <version>` for generating a secret
```

Comment:
You can use `abra app secret general --all` (or short `-a`) for generating all the remaining secrets, so set the "external" ones first, then can use all to set the generated ones.

Reply:
@nicksellen, there is one of the recently added secrets that did not accept the one generated with this command, but I don't remember which one. 🙈 Therefore, I would leave it out for now.

Reply:
Great, merged! One day I hope the secret management can know the difference between ones that have to be specifically inserted, and generated ones, ... and _how_ to generate specific secrets.